packages/x-codemod/src/v8.0.0/charts/preset-safe/actual.spec.tsx

```diff
@@ -7,9 +7,25 @@ import { ResponsiveChartContainer } from '@mui/x-charts/ResponsiveChartContainer
 import { ChartsOnAxisClickHandler } from '@mui/x-charts/ChartsOnAxisClickHandler';
 import { ChartsXAxis } from '@mui/x-charts/ChartsXAxis';
 import { LegendPosition } from '@mui/x-charts/ChartsLegend';
+import {
+  unstable_useSeries,
+  unstable_usePieSeries,
+  unstable_useLineSeries,
+  unstable_useBarSeries,
+  unstable_useScatterSeries,
+} from '@mui/x-charts/hooks';
+import { unstable_useHeatmapSeries } from '@mui/x-charts-pro/hooks';
 
-// prettier-ignore
-<div>
+function App() {
+  const series = unstable_useSeries();
+  const pieSeries = unstable_usePieSeries();
+  const lineSeries = unstable_useLineSeries();
+  const barSeries = unstable_useBarSeries();
+  const scatterSeries = unstable_useScatterSeries();
+  const heatmapSeries = unstable_useHeatmapSeries();
+
+  // prettier-ignore
```

Comment:
Do I need to keep `// prettier-ignore`?

Reply:
very likely that it needs to be there. We don't run prettier formatted on the output, so it is a bit finicky

Reply:
Yeah one day we will solve this prettier issue.

In brief, the `pnpm prettier` will pretiffy your tests. But the tested scrip will not so if you remove this comment you have the choice between

- the test static fails because the `actual.spec.tsx` is modified by `pnpm prettier`
- the test unit failse bacaus the result of the codemode does not match the expected one (because the script don't pretify it)

Removing prettier is not the cleanest, but it works 🙈

Reply:
Makes sense, thanks for the explanation 😄